reactor-core/src/test/java/reactor/core/publisher/FluxReplayTest.java

```diff
@@ -423,7 +501,7 @@ public void onlyInitialRequestWithLateUnboundedSubscriber() {
 		AssertSubscriber<Integer> ts = AssertSubscriber.create();
 		replay.subscribe(ts); //unbounded
 
-		assertThat(requests).containsExactly(5L, 8L);
+		assertThat(requests).containsExactly(3L, 3L);
```

Comment:
this test needs to be a bit more demonstrative too. if the source was capable of producing more elements, we should demonstrate that the early subscriber triggers a total of 5+8=13 requests, and yet the late subscriber still only sees the last 3 values.

Reply:
fixed